playbooks/configure_vm_host_network.yml

```diff
@@ -0,0 +1,12 @@
+---
+- import_playbook: playbooks/process_kvm_nodes.yml
```

Comment:
Try adding this to meta/main.yml. You wont need to import the process_kvm_nodes playbook.

```dependencies:
  - { role: process_kvm_nodes, transport_config: ... } ```